src/App.tsx

```diff
@@ -266,6 +312,27 @@ function App() {
         alert(`An error occurred ${e || "unknown"}`);
       });
   }, [keyFobId, getFobUserStats]);
+
+  const onAddressScanned = useCallback(async (address: string) => {
+    setQrCodeData(address);
+
+    // If this is Hype transaction, resolve HL Name (Primary Name)
+    if (nextPageState?.data?.chain?.name === "Hype") {
+      try {
+        const statsData = await fetch(`${ATM_BACKEND_URL}/stats`).then(x => x.json());
```

Comment:
leaving a comment for whoever gets it first - ?address={address} has to be included in the urn. resolvedHypeName is of address type